tests/baselines/reference/tsbuild/demo/initial-build/in-master-branch-with-everything-setup-correctly,-reports-no-error.js

```diff
@@ -0,0 +1,292 @@
+//// [/lib/initial-buildOutput.txt]

```

Comment:
commas are allowed in file names??? why didn't anybody tell me,,,,

Reply:
hmm they are but good catch i will remove commas... seems wierd